tests/upgrade_v2_script/static/fluent_bit_no_image.input.yaml

```diff
@@ -0,0 +1,155 @@
+      Name            systemd
+      Tag             host.*
+      DB              /tail-db/systemd-state-sumo.db
+      Systemd_Filter  _SYSTEMD_UNIT=addon-config.service
+      Systemd_Filter  _SYSTEMD_UNIT=addon-run.service
+      Systemd_Filter  _SYSTEMD_UNIT=cfn-etcd-environment.service
+      Systemd_Filter  _SYSTEMD_UNIT=cfn-signal.service
+      Systemd_Filter  _SYSTEMD_UNIT=clean-ca-certificates.service
+      Systemd_Filter  _SYSTEMD_UNIT=containerd.service
+      Systemd_Filter  _SYSTEMD_UNIT=coreos-metadata.service
+      Systemd_Filter  _SYSTEMD_UNIT=coreos-setup-environment.service
+      Systemd_Filter  _SYSTEMD_UNIT=coreos-tmpfiles.service
+      Systemd_Filter  _SYSTEMD_UNIT=dbus.service
+      Systemd_Filter  _SYSTEMD_UNIT=docker.service
+      Systemd_Filter  _SYSTEMD_UNIT=efs.service
+      Systemd_Filter  _SYSTEMD_UNIT=etcd-member.service
+      Systemd_Filter  _SYSTEMD_UNIT=etcd.service
+      Systemd_Filter  _SYSTEMD_UNIT=etcd2.service
+      Systemd_Filter  _SYSTEMD_UNIT=etcd3.service
+      Systemd_Filter  _SYSTEMD_UNIT=etcdadm-check.service
+      Systemd_Filter  _SYSTEMD_UNIT=etcdadm-reconfigure.service
+      Systemd_Filter  _SYSTEMD_UNIT=etcdadm-save.service
+      Systemd_Filter  _SYSTEMD_UNIT=etcdadm-update-status.service
+      Systemd_Filter  _SYSTEMD_UNIT=flanneld.service
+      Systemd_Filter  _SYSTEMD_UNIT=format-etcd2-volume.service
+      Systemd_Filter  _SYSTEMD_UNIT=kube-node-taint-and-uncordon.service
+      Systemd_Filter  _SYSTEMD_UNIT=kubelet.service
+      Systemd_Filter  _SYSTEMD_UNIT=ldconfig.service
+      Systemd_Filter  _SYSTEMD_UNIT=locksmithd.service
+      Systemd_Filter  _SYSTEMD_UNIT=logrotate.service
+      Systemd_Filter  _SYSTEMD_UNIT=lvm2-monitor.service
+      Systemd_Filter  _SYSTEMD_UNIT=mdmon.service
+      Systemd_Filter  _SYSTEMD_UNIT=nfs-idmapd.service
+      Systemd_Filter  _SYSTEMD_UNIT=nfs-mountd.service
+      Systemd_Filter  _SYSTEMD_UNIT=nfs-server.service
+      Systemd_Filter  _SYSTEMD_UNIT=nfs-utils.service
+      Systemd_Filter  _SYSTEMD_UNIT=node-problem-detector.service
+      Systemd_Filter  _SYSTEMD_UNIT=ntp.service
+      Systemd_Filter  _SYSTEMD_UNIT=oem-cloudinit.service
+      Systemd_Filter  _SYSTEMD_UNIT=rkt-gc.service
+      Systemd_Filter  _SYSTEMD_UNIT=rkt-metadata.service
+      Systemd_Filter  _SYSTEMD_UNIT=rpc-idmapd.service
+      Systemd_Filter  _SYSTEMD_UNIT=rpc-mountd.service
+      Systemd_Filter  _SYSTEMD_UNIT=rpc-statd.service
+      Systemd_Filter  _SYSTEMD_UNIT=rpcbind.service
+      Systemd_Filter  _SYSTEMD_UNIT=set-aws-environment.service
+      Systemd_Filter  _SYSTEMD_UNIT=system-cloudinit.service
+      Systemd_Filter  _SYSTEMD_UNIT=systemd-timesyncd.service
+      Systemd_Filter  _SYSTEMD_UNIT=update-ca-certificates.service
+      Systemd_Filter  _SYSTEMD_UNIT=user-cloudinit.service
+      Systemd_Filter  _SYSTEMD_UNIT=var-lib-etcd2.service
+      Max_Entries     1000
+      Read_From_Tail  true
+
+    @INCLUDE fluent-bit-output.conf
```

Comment:
Imho we should check only image migration, not full fluent-bit config

Reply:
Good point, I deleted most of the sections (left some of them).

Reply:
Also I added another testcase just for the image section.

Reply:
Done.